service/src/test/java/com/epam/rft/atsy/service/passwordchange/validation/impl/TestPasswordAllFieldFilledRuleForInvalidInputs.java

```diff
@@ -0,0 +1,89 @@
+package com.epam.rft.atsy.service.passwordchange.validation.impl;
+
+import com.epam.rft.atsy.service.domain.PasswordChangeDTO;
+import com.google.common.collect.*;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(Parameterized.class)
+public class TestPasswordAllFieldFilledRuleForInvalidInputs {
```

Comment:
The word "Test" should be at the end of the class name, to be consistent with other tests.